docs/COREDUMP_PACKAGE_METADATA.md

```diff
@@ -49,7 +49,7 @@ the original executable, debuginfo and sources, to further facilitate debugging.
 SECTION: `.note.package`
```

Comment:
While you are at it, maybe you can add that the section name is not significant?

Reply:
sorry, what do you mean by that?

Reply:
The name does not matter, consumers need to look at all note sections and parse the notes contained in them.

Reply:
But it matters for a human reader, no? Someone running `readelf --notes` should see `.note.package`, and not `.note.foobar.baz`. For tools there's the ID of course.

Reply:
I've added a line saying that when implementing parsers, the note should be searched using the id

Reply:
*node-id* should probably be *note type*? Also in the section description. But otherwise, works for me.

Reply:
thanks, fixed that too